whisperspeech/a2wav.py

```diff
@@ -50,3 +50,11 @@ def decode_to_notebook(self, atoks):
 
         audio = self.decode(atoks)
         display(Audio(audio.cpu().numpy(), rate=24000))
+
+    def decode_to_playback(self, atoks):
+        import sounddevice as sd
+        import numpy as np
```

Comment:
You don't need that line because you never refer to `np.` in the code below. Using methods on numpy arrays is possible even without importing the module.

Reply:
Didn't realize that...thanks!